aladdin-container.sh

```diff
@@ -91,33 +106,30 @@ function environment_init() {
 
     _handle_aws_config
 
+    if "$IS_LOCAL"; then
+        mkdir -p $HOME/.kube/
+        cp $HOME/.kube_local/config $HOME/.kube/config
+        # Replace e.g. /Users/ptr/.minikube with /root/.minikube in the minikube conf dir path
+        # (since that's where it will be mounted within the aladdin container)
+        sed 's|: \?.*\.minikube|: /root/.minikube| ; /: \/root\/.minikube/ s|\\|/|g' $HOME/.kube_local/config > $HOME/.kube/config
+    elif $INIT || ! test -f "$HOME/.kube_local/$CLUSTER_NAME.config"; then
+        echo "Getting cluster config using kops"
+        kops export kubecfg --name $CLUSTER_NAME > /dev/null
+        _add_authentication_users_to_kubeconfig
+        cp $HOME/.kube/config $HOME/.kube_local/$CLUSTER_NAME.config
+    else
+        echo "Using cached cluster config"
+        mkdir -p $HOME/.kube/
+        cp $HOME/.kube_local/$CLUSTER_NAME.config $HOME/.kube/config
+    fi
+    _handle_authentication_config
+    kubectl config use-context "$NAMESPACE.$CLUSTER_NAME"
```

Comment:
kubectl will act on `$HOME/.kube`, this is a directory that exists only on the container (not mounted on host or otherwise), so there won't be any crosstalk 👍